backend/tests/test_graphql_api/test_user/test_query_current_user.py

```diff
@@ -295,11 +295,15 @@ def test_query_current_user__unit_admin__unit_group(graphql):
 def test_query_current_user__unit_admin__multiple_roles(graphql):
     unit_1 = UnitFactory.create()
     unit_2 = UnitFactory.create()
+    unit_3 = UnitFactory.create()
+    unit_group = UnitGroupFactory.create(units=[unit_2, unit_3])
 
     user = UserFactory.create()
 
-    unit_role_1 = UnitRoleFactory.create(user=user, role=UserRoleChoice.RESERVER, units=[unit_1])
-    unit_role_2 = UnitRoleFactory.create(user=user, role=UserRoleChoice.VIEWER, units=[unit_2])
+    UnitRoleFactory.create(user=user, role=UserRoleChoice.RESERVER, units=[unit_1])
+    UnitRoleFactory.create(user=user, role=UserRoleChoice.VIEWER, units=[unit_1, unit_2])
+    UnitRoleFactory.create(user=user, role=UserRoleChoice.NOTIFICATION_MANAGER, units=[unit_2])
```

Comment:
Should be `HANDLER` also in this class, since `NOTIFICATION_MANAGER` should not be possible as a unit role.